django/db/models/sql/query.py

```diff
@@ -1263,6 +1263,8 @@ def _add_q(self, q_object, used_aliases, branch_negated=False,
                                          negated=q_object.negated)
         joinpromoter = JoinPromoter(q_object.connector, len(q_object.children), current_negated)
         for child in q_object.children:
+            if not isinstance(child, Node):
+                child = self._prepare_lookup(child)
```

Comment:
Is there a more specific name that `_prepare_lookup`? 

(I can't instantly come up with something ideal but it would be nice to not have to look at the implementation to see what this is for...)

Reply:
renamed to ``_expand_forward_related_filter``